### PR TITLE
Add schedule and keynote

### PR DESCRIPTION
**Pull request by lucaspinto**
This PR adds the missing documentation for the MC design, simplifies the schedule design and add a design for the keynote

### CODE REVIEW

**Comment by hagenburger** [comment]
Keynote changes are fine. Can you add the schedule to the style guide so I can have a look?

**Comment by lucaspinto** [comment]
@hagenburger you're right! Taking care of this this evening.

**Comment by lucaspinto** [comment]
Hey @hagenburger, I've add the LSG for the schedule.

**Comment by lucaspinto** [comment]
Hey @hagenburger, I made the day name bigger as we said yesterday. It's now 5rem and centered. IMHO it's good like this.

File: source/2015/components/_layout.haml.orig [hunk 1]
```diff
@@ -0,0 +1,36 @@
+:ruby
+  current_website = "http://2015.eurucamp.org"
+  header = {
+    root_path: current_website,
+    items: [
+      { url: "http://tickets.eurucamp.org/", text: "Tickets" },
+<<<<<<< HEAD
```
Comment (hagenburger) [inline source/2015/components/_layout.haml.orig]: Please remove this file :)

**Comment by hagenburger** [comment]
This is was merged in #29.

**Pull request closed without merging**